spec/controllers/v0/feature_toggles_controller_spec.rb

```diff
@@ -50,29 +50,6 @@
       expect(json_data['data']['features']).to include({ 'name' => @feature_name_camel, 'value' => true })
       expect(json_data['data']['features']).to include({ 'name' => @feature_name, 'value' => true })
     end
-
-    context 'when flipper.mute_logs settings is true' do
```

Comment:
Since there's only 1 SQL query, I removed the ability to mute the logs as it's not spammy anymore.